### PR TITLE
readme: fix steps list nesting

### PR DESCRIPTION
**Pull request by marcospb19**
Fixes the numbered list nesting not working.

### CODE REVIEW

**Comment by marcospb19** [comment]
Here's a link to the rich diff:

https://github.com/bevyengine/bevy-website/pull/145/files?short_path=b335630#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5

As we can see, the markdown table rendered in GitHub was not nested as intended in the file.
image

File: README.md [hunk 1]
```diff
@@ -4,14 +4,13 @@ The source files for https://bevyengine.org. This includes official Bevy news an
...
-The Bevy website is built using the Zola static site engine. In our experience, it is fast, flexible, and straightforward to use. [Check it out here](https://www.getzola.org/)!
+The Bevy website is built using the Zola static site engine. In our experience, it is fast, flexible, and straightforward to use.
 
 To check out any local changes you've made:
-1. Download Zola.
-2. Clone the Bevy Website git repo and move to that directory:
-  1. `git clone https://github.com/bevyengine/bevy-website.git`
-  2. `cd bevy-website`
-3. Start the Zola server with `zola serve`.
+- [Download Zola](https://www.getzola.org/).
```
Comment (marcospb19) [inline README.md:10]: In addition of fixing the nested list, I removed the "Check it out" link and added it to the list itself.

**Review by alice-i-cecile — APPROVED** [review]

**Comment by alice-i-cecile** [comment]
@cart this has my unofficial `ready-for-cart` label (since it doesn't exist here) ;)

**Comment by cart** [comment]
I think this should still be an ordered list. Fixing the sub-items just required more indentation.

**Comment by marcospb19** [comment]
Nice, I didn't knew about the roman numerals :sparkles:.

image

---

Same instructions can be found at https://bevyengine.org/learn/book/contributing/docs/:

image

I think it's good too, and does not require changing, just sending here in case someone disagrees.

**Pull request merged**